CHANGELOG.md

```diff
@@ -3,16 +3,86 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+[#446](https://github.com/plotly/dash-table/pull/446)
+- Table API rework
+#### NEW
+    - `column.sort_as_null`: Allows sorting behavior customization.
+        Accepts an array of string, number or booleans.
+
+#### REMOVED
+    - `column.clearable`: Allows clearing the value of a dropdown cell.
+        Removed in favor of `dropdown_**` `clearable` nested property
+    - `column.options`
+        Removed. Redundant with `dropdown`.
+    - `pagination_settings`
+        Replaced by two props `page_current` and `page_size`.
+
+#### RENAMED
+    - `column_static_tooltip`
+        Renamed `tooltip`.
+    - `column_conditional_tooltips`
+        Renamed `tooltip_conditional`.
+    - `filter`
+        Renamed `filter_query`.
+    - `sort_type`
+        Renamed `sort_mode`.
+    - `derived_filter_structure`
+        Renamed to `derived_filter_query_structure`.
+
+#### MODIFIED
+    - `column.deletable`: Allows column deletion.
+        Now accepts a boolean or an array of booleans (for multi-line headers)
+    - `column.editable_name`: Allows column renaming.
+        Renamed to `column.renamable`
+        Now accepts a boolean or an array of booleans (for multi-line headers)
+    - `column.id`
+        Now accepts `string` only -- `number` column ids can be casted to string
+    - `n_fixed_columns`: Will fix columns to the left.
+        Renamed to `fixed_columns`
+        Now accepts an object { headers: boolean, data: number } instead of a number.
+        { headers: true } determines the number of columns to fix automatically.
+    - `n_fixed_rows`: Will fix rows to the top.
+        Renamed to `fixed_rows`
+        Now accepts an object { headers: boolean, data: number } instead of a number.
+        { headers: true } determines the number of rows to fix automatically.
```

Comment:
```suggestion
        { headers: true } determines the number of rows to fix automatically (i.e. if there are multiple headers, it will fix all of them as well as the filter row).
        { headers: true, data: 2} would fix all of the header rows as well as the first 2 data rows.
```